Dockerfile

```
@@ -0,0 +1,8 @@
+FROM python:3.11
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY . .
+
+CMD [ "python3", "-m" , "flask", "run", "--host=0.0.0.0"]
```

It's likely that you'd also need a CMD to flask db upgrade for any new migrations that came along since the last time the container was run.
```suggestion
CMD [ "python3", "-m" , "flask", "db", "upgrade"]
CMD [ "python3", "-m" , "flask", "run", "--host=0.0.0.0"]
```